pkg/apis/datadoghq/v1alpha1/datadogagent_types.go

```diff
@@ -664,6 +664,11 @@ type ExternalMetricsConfig struct {
 	// +optional
 	Enabled bool `json:"enabled,omitempty"`
 
+	// Enable informer and controller of the watermark pod autoscaler
+	// NOTE: You need to install the `WatermarkPodAutoscaler` CRD before
```

Comment:
maybe something more generic: the WatermarkPodAutoscaler-Controller (https://github.com/DataDog/watermarkpodautoscaler) need to be install

Reply:
good point, done 👍